.github/actions/support-package-metadata/action.yml

```diff
@@ -0,0 +1,39 @@
+name: Create support-packages metadata
+description: |
+  Creates all required metadata for building support-packages.
+inputs:
+  released-versions-file:
+    default: ${{ github.workspace }}/RELEASED_VERSIONS
+    description: |
+      File containing the mapping of collector to stackrox versions
+  metadata-directory:
+    default: /tmp/support-packages/metadata/
+    description: |
+      Output directory for generated metadata
+  branch-name:
+    required: true
+    description: |
+      Branch CI is running on
+
+runs:
+  using: composite
+  steps:
+  - name: Map collector to stackrox versions
+    shell: bash
+    run: |
+      ${{ github.workspace }}/kernel-modules/support-packages/01-collector-to-rox-version-map.py \
+      ${{ inputs.released-versions-file }} \
+      ${{ inputs.metadata-directory }}
+
+  - name: Create driver version metadata
+    shell: bash
+    env:
+      BRANCH_NAME: ${{ inputs.branch-name }}
+    run: |
+      ${{ github.action_path }}/create-metadata-dir.sh
```

Comment:
If 02-fetch-collectors-metadata.sh is no longer being used, it should probably be deleted and other associated changes should be made. I.e update README and run-all.sh in kernel-modules/support-packages.

Reply:
I created `run-all.sh` when working with OSCI, but the intention was that it could be run outside of OSCI. The intention was that it could be run manually or in another CI. I am not requiring that GHA use it. I think we did discuss that possibility earlier and decided not to do that. This issue can be dealt with in another PR.